addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -351,6 +352,158 @@ var MassMailingFieldHtml = FieldHtml.extend({
             'min-height': '100vh',
             width: '100%'
         });
+
+        this._initDynamicPlaceholderGenerator();
+    },
+
+    /**
+     * @private
+     */
+    _initDynamicPlaceholderGenerator: function () {
+        const getCommandsForSearchTerm = async searchTerm => {
+            const searchData = await rpc.query({
```

Comment:
Normalement ici tu devrais pouvoir utiliser `this._rpc` non ? (ça s'applique aux autres aussi évidemment)
```js
this._rpc({
    model: ...,
    method: ...,
    kwargs: ...,
}
```